### ISSUE TITLE
Cannot delete a page that is Published=No 

### ISSUE THREAD

**Issue by Ribero** [getgrav/grav-plugin-admin#2031]
Grav v1.7.1 - Admin v1.10.1

- Create a new page in /root using Admin Panel GUI
- Change status of page from Published=Yes (default) to Published=No -> you'll get an error (see #2030) but the page is then marked as non published and you can verify it on the site
- From Admin Panel GUI select the page and delete it:
<img width="407" alt="image" src="https://user-images.githubusercontent.com/2266264/105385746-f89b6680-5c13-11eb-9823-2f08d7333376.png">
You can see that even if the page has the red dot (not published)  the menu shows the contrary (green lable Published)
- select delete
- You will see that the page as not been deleted but it now has a green dot (published) and you can effectively see it's entry on the site menu but you will get a 404 if you click it
- You can then try a second delete and this time the page will be deleted and the web site menu updated.

**Comment by mahagr** [comment]
What happens is that the page gets deleted, but the folder remains. This causes the page to become a folder, so what you see is expected.

I think this happens only with multi-language functionality turned on, it somehow fails to detect that the page does not exist in any language.

**Comment by beamaria** [comment]
Same problem with me with a bilingual site (english /irish)

**Comment by mahagr** [comment]
I think that this has already been fixed while fixing another issue. As I cannot reproduce it anymore.

**Comment by realyussel** [comment]
I have a same problem, with a site only in spanish.

**Comment by mahagr** [comment]
I hope this will be fixed after the next release goes out.

**Comment by mahagr** [comment]
Should be fixed?

**Comment by Eihrister** [comment]
Hello!

I ran into this issue earlier, but whether or not a page is Published does not matter.

I have a multilingual site, and it seems that if the last language file (default.en.md for instance) in the directory has been deleted, it fails to detect that it was the last file, and thus does not clean up the directory/folder that's left behind. The name shown in the admin panel then becomes the slug (directory name). Deleting it again does remove the entry, because then it sees the directory is empty.

Whether it's deleted from the Pages view or clicking the Delete button when having the specific page open, does not make any difference. Cache on or off also makes no difference. Leaving 1 single language configured or having 4 (en, nl, pt, and es in my case) does not matter either.

The main issue seems to be:

> Failure to detect that the directory/folder is empty after deleting the last remaining language-specific file (`default.XX.md`, for instance), which results in not cleaning it up after.

**Comment by mahagr** [comment]
You're right. I just checked and the delete only deletes the current language unless you delete the "languageless" version of the page.
